.github/workflows/build.yml

```diff
@@ -79,5 +79,5 @@ jobs:
     - name: make baresip macOS
       if: ${{ runner.os == 'macOS' }}
       run: |
-        make V=1 CCACHE= info test modules
-        make clean; make CCACHE= STATIC=yes
+        make V=1 CCACHE= SYSROOT_LOCAL=/usr/local info info2 test modules
+        make clean; make CCACHE= SYSROOT_LOCAL=/usr/local STATIC=yes
```

Comment:
macOS should not require any special args to build. A simple `make` should be sufficient to build baresip on all supported platforms.

Reply:
But if one specifies SYSROOT for a cross compilation build then it is not common to include /usr/local into the header search paths.
e.g. yocto says:
```
cc1: warning: include location "/usr/local/include/rem" is unsafe for cross-compilation [-Wpoison-system-directories]
```

Reply:
SYSROOT for macOS is defined here https://github.com/baresip/re/blob/master/mk/re.mk#L23

Reply:
Now I added /usr/local as fallback. Also no warning/error in our yocto build.